### PR TITLE
Fix/william/rel57/ee tls

### PR DESCRIPTION
**Pull request by qzhuyan**
Fixes [EMQX-12417](https://emqx.atlassian.net/browse/EMQX-12417)
replacing https://github.com/emqx/emqx/pull/13128

Release version: v/e5.?

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [x] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

[EMQX-12417]: https://emqx.atlassian.net/browse/EMQX-12417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/emqx_auth_ext/test/emqx_auth_ext_test_tls_certs_helper.erl [hunk 1]
```diff
@@ -0,0 +1,307 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_auth_ext_test_tls_certs_helper).
```
Comment (zmstone) [inline apps/emqx_auth_ext/test/emqx_auth_ext_test_tls_certs_helper.erl:5]: can this module reside in the OSS part?
at least the gen_ functions can be useful there too ?

File: rel/i18n/emqx_schema.hocon [hunk 2]
```diff
@@ -684,6 +684,7 @@ common_ssl_opts_schema_verify.desc:
 common_ssl_opts_schema_verify.label:
 """Verify peer"""
 
+
```
Comment (zmstone) [inline rel/i18n/emqx_schema.hocon:687]: nit:
```suggestion
```

File: apps/emqx_auth_ext/test/emqx_auth_ext_test_tls_certs_helper.erl (hunk 1 above)
Comment (qzhuyan) [reply apps/emqx_auth_ext/test/emqx_auth_ext_test_tls_certs_helper.erl:5]: Is there any known method to make this app find the test helper module resides in another app while running common test?

I only know this way: 
https://github.com/emqx/emqtt/blob/master/test/emqtt_test_lib.erl#L58-L70

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-2596)

**Pull request merged**